docs/building-spark.md

```diff
@@ -12,7 +12,7 @@ redirect_from: "building-with-maven.html"
 ## Apache Maven
 
 The Maven-based build is the build of reference for Apache Spark.
-Building Spark using Maven requires Maven 3.3.9 or newer and Java 8+.
+Building Spark using Maven requires Maven 3.3.9 or newer and Java 8.
```

Comment:
Ur, @kiszk and @cloud-fan , @HyukjinKwon .

It seems to be Maven version is also outdated. We updated Maven version in  https://github.com/apache/spark/pull/21905 and https://github.com/apache/spark/pull/21920 .

Should we change it together? Or, Maven 3.3.9 is still valid to compile Spark?

Reply:
let's change to the expected maven version, thanks!

Reply:
+1

Reply:
Thank you, @cloud-fan and @HyukjinKwon .